### PR TITLE
Remove type and branding package reference

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes 4864 .

#### Changes proposed in this Pull Request:

- Remove Branding package reference
- Remove Types package reference
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- Please run yarn precommit 

#### Any specific feedback you are looking for?

Note: For Readme and Migration Update will raise different PR.

### CODE REVIEW

**Comment by joebochill** [comment]
We should be able to eliminate the /ui subfolder now that there are not multiple sub-packages...I think we can move everything up to the root.

I would like to see the usage look like (feel free to offer alternative suggestions if you think there's a better way):

```tsx
import Colors, { BrandingColors } from '@brightlayer-ui/colors';
```
or
```tsx
import Colors from '@brightlayer-ui/colors';
import BrandingColors from '@brightlayer-ui/colors/branding';
```

(Small thing) — I'd also like to remove the specific references to Angular/React/React Native/Etc. and instead refer to usage for "JSON Object" and/or "CSS variables"

**Comment by manojleaton** [comment]
Screenshot 2023-11-09 at 12 20 45 PM

Please check this

**Comment by manojleaton** [comment]
Looks good!

File: README.md [hunk 1]
```diff
@@ -1,104 +1,86 @@
...
-background-color: map-get(palette.$blui-blue, 500)
+background-color: map-get($blui-primary, 10);
 ```
 
-### React or React Native
-
 ```
-import * as Colors from '@brightlayer-ui/colors';
+// For Branding Colors in styles.scss or your top-level sass file
+@import '~@brightlayer-ui/colors/branding-pallete.scss';
```
Comment (joebochill) [inline README.md:74]:
```suggestion
@import '~@brightlayer-ui/colors/branding-palette.scss';
```
Make sure the file name isn't spelled incorrectly

File: README.md [hunk 2]
```diff
@@ -1,104 +1,86 @@
 # Brightlayer UI Colors
 
-This repository contains definitions for various colors used throughout Brightlayer UI applications.
+This package contains Brightlayer UI color definitions. These are intended for use as primary UI colors.
```
Comment (joebochill) [inline README.md:3]: I prefer the older intro wording:
```suggestion
This repository contains definitions for various colors used throughout Brightlayer UI applications.
```

**Comment by joebochill** [comment]
Folder structure can be simplified...we don't need ui and branding folders with single files in them...we can just have a flat structure with the files names appropriately.

**Comment by joebochill** [comment]
I'd still like to see default exports being used so users aren't forced to use our variable names. Having the actual palette as a sub-item under the default export is also bizarre.

```tsx
import Colors from '@brightlayer-ui/colors';
import BrandingColors from '@brightlayer-ui/colors/branding';

const myColor = Colors.primary['10']; // not Colors.BLUIColors.primary['10']
```

If you want to provide a named export for the branding colors at the root, this is also fine:
```tsx
import Colors, { BrandingColors } from '@brightlayer-ui/colors';
import DefaultExportBrandingColors from '@brightlayer-ui/colors/branding'; // should still work

const equal = (BrandingColors['teal'] === DefaultExportBrandingColors['teal']); // true
```

File: .github/dependabot.yml [hunk 3]
```diff
@@ -8,25 +8,4 @@ updates:
...
-      schedule:
-          interval: 'monthly'
-          day: 'monday'
-      open-pull-requests-limit: 1
-      target-branch: 'dev'
-      labels:
-          - 'external-dependency'
-          
\ No newline at end of file
+          - 'external-dependency'
```
Comment (ektaghag-eaton) [inline .github/dependabot.yml:11]: It is there, I deleted branding folder

Comment (joebochill) [reply .github/dependabot.yml:11]: Got it, I wasn't seeing the entire file in the diff

**Review by joebochill — APPROVED** [review]

**Pull request merged**